src/encord_active/app/actions_page/export_filter.py

```diff
@@ -482,7 +482,7 @@ def render_export_button(
     disabled_help = None
     if not action_utils:
         disabled_help = "Contact Encord to enable integration"
-    elif not is_filtered:
+    elif is_filtered:
```

Comment:
discovered when verifying export works. Filter was switched, export to encord button only worked when a filter was selected. This is different to the error message.